draft-ietf-scitt-architecture.md

```diff
@@ -666,6 +666,13 @@ COSE_Sign1 = [
   payload : bstr,
   signature : bstr
 ]
+<!-- https://datatracker.ietf.org/doc/draft-ietf-cose-cwt-claims-in-headers/ -->
+<!-- https://www.iana.org/assignments/cwt/cwt.xhtml -->
+CWT_Claims = {
+  1 => tstr; iss, the issuer that is making statements
+  2 -> tstr; sub, the subject about which the statements are made, throughout this spec, this is also called feed.
```

Comment:
I was going to suggest removing:
> , throughout this spec, this is also called feed.

and use a separate PR to replace Feed with Subject. 
To keep this PR isolated to these changes, I'd leave this sentence and cleanup in another PR: Update Feed to be Subject